index.html

```diff
@@ -11,9 +11,223 @@
+                recusandae quam laboriosam adipisci natus laudantium impedit quis?
+              </p>
+            </div>
+          </li>
+          <li class="speaker-item d-flex">
+            <img src="images/Wilhelm.png" style="width: 30%; height: 190px" alt="Alfred Adler">
+            <div>
+              <h6>Zeenat Lawal</h6>
+              <small>University of Ilorin, Ilorin, Kwara State</small>
+              <p class="line"></p>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem dolor unde, 
+                consectetur,illum, sed accusamus maiores placeat sit molestiae commodi deserunteos 
+                recusandae quam laboriosam adipisci natus laudantium impedit quis?
+              </p>
+            </div>
+          </li>
+        </div>        
+      </ul>
+    </section> -->
+
+    <!-- MORE INFORMATION SECTION -->
+    <!-- <img src="images/" alt="">
+
+    <section class="more-info">
+      <div>
+        <h1>
+          PSYCH AND MENTAL HEALTH | SPOKEN WORDS CONFERENCE 2021
+        </h1>
+        <p class="more-info-detail"> This is an bi-annual Psychology Conference that 
+          brings togther prominent Psychologists in the world, to share knowledge,
+          tackle problems and seek solutions to impending dangers of mental health.
+          This year's conference would be in a conducive environment of Sheraton
+          Hotels, Abuja, Nigeria. It would hold for 3days, from 17th - 20th of October, 2021.
+          For this year, we'll be expecting a host of professional in the field of
+          Mental Health and there would discussion time, where we socialize and learn.</p>
+      </div>
+      <div class="email-us">
+        <p>For more information on this event, <br /> please contact the helpdesk!</p>
+        <a href="mailto:nnaemekasilver@gmail.com">nnaemekasilver@gmail.com</a>
+      </div>
+    </section> -->
+    <section class="partners text-center">
+      <h1 class="pt-4 partners-heading">Partner</h1>
+      <ul class="partners-list d-flex flex-row justify-content-between m-5">
+        <li>
+          <img class="img-fluid" src="images/google.jpg" alt="Google Partner">
+        </li>
+        <li>
+          <img class="img-fluid" src="images/airbnb.png" alt="AirBnB Partner">
+        </li>
+        <li>
+          <img class="img-fluid" src="images/amazon.png" alt="Amazon Partner">
+        </li>
+        <li>
+          <img class="img-fluid" src="images/facebook.png" alt="Facebook Parner">
+        </li>
+      </ul>
+    </section>
+  </main>
+   <footer class="d-flex gap-4 fs-6 p-3">
+     <div class="footer-logo">
+      <img src="images/psychology-logo.jpg" alt="Psychology Logo" width="50" height="50" class="d-inline-block align-text-top">
+    </div>
+    <div class="rights">
+      <p class="fw-bold text-center">2021 Psych Conference | Health <br /> Some Rights Reserved</p>
+    </div>
+   </footer>
```

Comment:
- [ ]  Footer logo should be aligned to the left and text aligned to the right.
- [ ] On mobile screen sizes, the footer should have a white background, based on what we have in the original design.
- [ ] Also on desktop sizes, It should have a dark background.

Reply:
Well done working on these changes as well. However, please check my **OPTIONAL** inline comment under the review 😄